app/views/articles/_bottom_content.html.erb

```diff
@@ -5,9 +5,11 @@
       <a href="<%= article.path %>" data-preload-image="<%= cloud_cover_url(article.main_image) %>" class="crayons-link mt-6 block">
         <div class="flex items-center">
           <span class="crayons-avatar crayons-avatar--xl m:crayons-avatar--2xl mr-4 shrink-0">
-            <%= optimized_image_tag(article.cached_user.profile_image_url,
-                                    optimizer_options: { crop: "imagga_scale", width: 100, height: 100 },
-                                    image_options: { loading: "lazy", alt: "#{article.cached_user.username} profile image", class: "crayons-avatar__image" }) %>
+            <% if article.cached_user.profile_image_url %>
+              <%= optimized_image_tag(article.cached_user.profile_image_url,
+                                      optimizer_options: { crop: "imagga_scale", width: 100, height: 100 },
+                                      image_options: { loading: "lazy", alt: "#{article.cached_user.username} profile image", class: "crayons-avatar__image" }) %>
+            <% end %>
```

Comment:
It's very possible that I just have some wonky seed data in my dev environment, but I found this throwing unhandled exceptions (`optimized_image_tag` does _not_ like `nil`) when previewing an article, and the logic for "when does a new article route to preview (instead of just rendering?)" is just a _bit_ convoluted to reliably stage ... and this prevents the error? (any `nil` profile images show up as blank black circles, which seems like the background leaking through, but might be better than an exception?)